charts/prefect-orion/Chart.yaml

```diff
@@ -7,7 +7,7 @@ dependencies:
       - bitnami-common
     version: 2.2.1
   - name: postgresql
-    condition: postgresql.useSubChart
+    condition: postgresql.enabled,postgresql.useSubChart
```

Comment:
From the docs:

>Condition - The condition field holds one or more YAML paths (delimited by commas). If this path exists in the top parent's values and resolves to a boolean value, the chart will be enabled or disabled based on that boolean value. Only the first valid path found in the list is evaluated and if no paths exist then the condition has no effect.

https://helm.sh/docs/topics/charts/#tags-and-condition-fields-in-dependencies

I tested this by using `helm template orion`, before:

```
$ helm template orion . --set=postgresql.enabled=false | grep postgres
# Source: prefect-orion/charts/postgresql/templates/secrets.yaml
  name: orion-postgresql
    app.kubernetes.io/name: postgresql
    helm.sh/chart: postgresql-11.9.13
  postgres-password: "RVFhYTVjM1ZnSg==
```

After, the output is empty.

Reply:
Do we need to still have the `useSubChart` value? or is its purpose duplicated?

Reply:
ah i see - this makes sense to me!

Reply:
@jawnsy I'm not sure this will work as expected. As you note - the condition only evaluates the `first` valid yaml path.  So if users enable `postgres` but do not want to use the `subchart`, this logic will not work for them

Reply:
@jamiezieziula ah, good catch! you read the docs more closely than I did.

any thoughts what we should do to handle this? the condition field doesn't seem to support an "and" operator, as far as I can tell

Reply:
I think we should move this logic elsewhere and leave `postgersql.useSubChart` as the condition on the chart itself. Let me take a look and get back to you!